developer-guide/concepts-and-designs/multicluster/architecture.md

```diff
@@ -0,0 +1,54 @@
+# Multiple Cluster Architecture
+
+## Introduction
+
+Today, it's very common for organizations to run a handful of Kubernetes Clusters. Each kubernetes cluster is a relatively self-contained unit. And there are several teams working on tying all clusters, such as kubefed. More and more KubeSphere users and customers have expressed their kneed interests in managing all clusters through KubeSphere.
+
+## Summary
+For KubeSphere users and customers, the two highest priority ones are relate to Operating clusters in one place and High Availability.
+
+## What is KubeSphere multiple cluster?
+KubeSphere multiple cluster is the ability to operate clusters in one place, which is KubeSphere, and extend kubefed to achieve High Availability.
+
+## Architecture
+The overall architecture of a control plane is shown as following.
+
+![](architecture.png)
+
+Some design principles we are following in this architecture.
+
+1. Keep the underlying KubeSphere clusters independent. The should have no knowledge of control plane or of each other.
```

Comment:
The should have no knowledge => They should have no knowledg

Reply:
fixed